frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -0,0 +1,48 @@
+@gitOps
+Feature: Applications managed by GitOps and Create secret
+    User should be able to see list of applications managed by GitOps
+
+
+        Background:
+            Given user is at developer perspective
```

Comment:
We need to add `user has installed gitops operator` in the background

Reply:
It included in the beforeAll() hook, so that no need to run install gitops operator on every feature file.
Even if we run single feature file, it works

Reply:
Sorry i will update this, As we have manual scenarios. the step makes more sense